### DISCUSSION TITLE
turborepo with nestjs ?

### DISCUSSION THREAD

**Discussion by orenmizr** [vercel/turborepo#438] [Help] [answered]
something like nx's dep managements and apps folder. can it work ?

**Comment by orenmizr** [comment]
do the maintainers know if it is on the roadmap ? no mention of it in the docs @jaredpalmer

**Reply by orenmizr** [reply]
⚰

**Reply by graup** [reply, marked as answer]
What exactly is your question? An `apps` folder works fine. Turbo doesn't care what's inside your packages. Did you encounter any problem with nestjs?
